data/equiv.lean

```diff
@@ -127,6 +127,24 @@ theorem apply_eq_iff_eq_inverse_apply : ∀ (f : α ≃ β) (x : α) (y : β), f
   ⟨λ e : f₁ x = y, e ▸ (l₁ x).symm,
    λ e : x = g₁ y, e.symm ▸ r₁ y⟩
 
+theorem left_inverse (f : equiv α β) : left_inverse f.symm f := f.left_inv
+
+theorem right_inverse (f : equiv α β) : function.right_inverse f.symm f := f.right_inv
```

Comment:
`right_inverse_symm` and why is it `function.right_inverse` instead of just `right_inverse`?

Reply:
Lean core has `right_inverse` defined in `logic.lean` and `function.right_inverse` defined in `function.lean`. Hence opening the `function` namespace and using `right_inverse` is ambiguous.

Reply:
Ah, okay I see.